.husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+# Run linter on staged files before commit
+
+STAGED_FILES=$(git diff --cached --name-only --diff-filter=ACMR | grep -E '\.(ts|js)$' | sed 's| |\\ |g')
+
+if [ -n "$STAGED_FILES" ]; then
+  npx eslint --fix $STAGED_FILES
+  git update-index --again
```

Comment:
@Cerchie From my understanding, these two lines auto-fix potential errors and stage the changes, so that the pre-commit would never fail. Is this assumption correct?